HW04/Igor_Hrybinyk/my_methods.rb

```diff
@@ -0,0 +1,39 @@
+class Array
+  def my_each
+    return dup.to_enum unless block_given?
```

Comment:
Why do you need `dup` here? 🤔 
```suggestion
    return to_enum unless block_given?
```

Reply:
It's my mistake, `dup` doesn't need here. Fixed.